### PR TITLE
Ds 218 configuration

### PR DESCRIPTION
**Pull request by dataswifty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,17 +76,17 @@ To launch the HAT, follow these steps:
     ```
 2. Compile the project:
     ```bash
-    > sbt compile
+    > make dev
     ```
 3. Add custom local domain mapping to your `/etc/hosts` file. This will make sure when you go to the defined address from your machine you will be pointed back to your own machine. E.g.:
     ```
     127.0.0.1   bobtheplumber.hat.org
```
Comment (leymytel) [inline README.md:83]: They probably need to add the "bobtheplumber.example.com" in hosts, right?

File: README.md [hunk 2]
```diff
@@ -76,17 +76,17 @@ To launch the HAT, follow these steps:
...
     ```
     127.0.0.1   bobtheplumber.hat.org
     ```
 4. Run the project:
     ```bash
-    > sbt "project hat" "run -Dconfig.resource=dev.conf"
+    > make run-dev
     ```
-5. Go to [http://bobtheplumber.hat.org:9000](http://bobtheplumber.hat.org:9000)
+5. Go to [http://bobtheplumber.exmaple.com:9000](http://bobtheplumber.example.com:9000)
```
Comment (leymytel) [inline README.md:89]: typo here!

File: README.md (hunk 1 above)
Comment (dataswifty) [reply README.md:83]: Fixed in the latest.  Thank you!

File: README.md (hunk 2 above)
Comment (dataswifty) [reply README.md]: Fixed in the latest.  Thank you!

File: build.sbt [hunk 3]
```diff
@@ -74,7 +74,7 @@ lazy val hat = project
   .settings(
     // Use the alternative "Ash" script for running compiled project form inside Alpine-derived container
     // as Bash is incompatible with Alpine
-    javaOptions in Universal ++= Seq("-Dapplication.mode=DEV", "-Denv=stage", "-Dhttp.port=9000", "-Dhttps.port=9001", "-Dpidfile.path=/dev/null", "-Dplay.server.pidfile.path=/dev/null", "-Dconfig.resource=docker.conf"),
+    javaOptions in Universal ++= Seq(""),
```
Comment (augustinas) [inline build.sbt:77]: Should we keep docker-specific `"-Dpidfile.path=/dev/null", "-Dplay.server.pidfile.path=/dev/null"` options in this list as they are being loaded only for container builds?

Comment (dataswifty) [reply build.sbt:77]: That seems reasonable.

**Pull request merged**